docs/Getting-started.md

```diff
@@ -0,0 +1,119 @@
+
+[order]: # (2)
+# Getting Started
+
+## Credentials
+Before moving on to the actual integration make sure you have the following credentials from Glia:
+* **App token**
+* **Site ID**
+
+You will also need to provide the **Visitor Context**. Visitor context specifies a content that can be shown to an Operator during an Engagement on the place of CoBrowsing section in Operator App. The only supported type for now is `.page`, which accepts a URL of a web page.
+
+## Glia Hub Setup
+Make sure you have the following information from the Glia Hub:
+* The **region** ("environment") in which your app will be located (Europe or USA).
+* The SDK will require using queues, where one or more operators can engage with customers. You will require a **Queue ID** in order for the app to understand which queue is should use.
+
+## Integration
+
+### Adding GliaWidgets iOS SDK to the Project
+
+#### Swift Package Manager (SPM)
+To integrate the GliaWidgets iOS SDK into your project via [Swift Package Manager](https://swift.org/package-manager/):
+1. In Xcode, go to File > Swift Packages > Add Package Dependency...
+2. Select your project in the opened menu.
+3. Enter GliaWidgets repository URL into the search field:
+`https://github.com/salemove/ios-sdk-widgets`
+
+#### CocoaPods
+To integrate the GliaWidgets iOS SDK into your project via [CocoaPods](https://cocoapods.org/) modify your `Podfile`:
+```
+source 'https://github.com/CocoaPods/Specs.git'
+source 'https://github.com/salemove/glia-ios-podspecs.git'
+
+user_frameworks!
+
+target 'YOUR_TARGET_NAME' do
+    pod 'GliaWidgets', git: 'https://github.com/salemove/ios-sdk-widgets', tag: 'SDK_VERSION'
```

Comment:
The user does not need to use the direct git link anymore :)
So we could update this line to
```suggestion
    pod 'GliaWidgets', 'SDK_VERSION'
```